evalml/tests/automl_tests/test_automl.py

```diff
@@ -4488,6 +4460,87 @@ def test_cv_validation_scores_time_series(
     assert cv_vals[0] == validation_vals[0]
 
 
+@pytest.mark.parametrize("algorithm,batches", [("iterative", 2), ("default", 3)])
+@pytest.mark.parametrize(
+    "parameter,expected",
+    [
+        ("mean", ["mean", "median", "most_frequent"]),
+        (Categorical(["mean"]), Categorical(["mean"])),
+    ],
+)
+@pytest.mark.parametrize("problem_type", ["binary", "time series binary"])
+def test_search_parameters_held_automl(
```

Comment:
Thoughts on adding a test to check whether the estimator parameters used match the default values? I think e can use `inspect` to make sure the test does not get stale. We chose the default values precisely cause they're fast/get good enough performance so might be good to add that coverage. And I think that unit test would have caught the original problem with the first version of this pr right?

Reply:
I like this idea!

Reply:
yep, can do!